src/common.rs

```diff
@@ -6,16 +6,21 @@ pub enum Architecture {
     Unknown,
     Aarch64,
     Arm,
+    Avr,
+    Bpf,
```

Comment:
I think this should be called `Ebpf`. BPF is a kind of precursor to, but not compatible with eBPF. I don't think BPF used ELF as executable format.

Reply:
This is true, but I think target triples used `bpf`.